src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -2016,6 +2016,23 @@ public void testNewHandler() throws Exception {
         verifyWarns(checkConfig, getPath("InputIndentationNewHandler.java"), expected);
     }
 
+    @Test
+    public void testChainedMethodWithBracketOnNewLine() throws Exception {
```

Comment:
please add test to https://github.com/checkstyle/checkstyle/tree/8c9a446fa86e6954f931ed4f366c36181997384c/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation
we usually almost copy Inputs from UTs (with corrections of expected), but make sure them are executed on google config.